client/src/partials/reports/modals/balance.config.js

```diff
@@ -25,7 +25,7 @@ function BalanceReportConfigController($state, ModalInstance, Notify, Languages,
     { number : 6, name : 'ACCOUNT.COST' },
     { number : 7, name : 'ACCOUNT.REV' },
     { number : 8, name : 'ACCOUNT.EXP_PROD' },
-    { number : '*', name : 'ACCOUNT.ALL_ACCOUNT' }
+    { number : '*', name : 'ACCOUNT.ALL_CLASSES' }
```

Comment:
I know you didn't write this code, but in the future, we should put these "Account Classes" in the `bhConstants` or just infer them from the first level title accounts.